module/zcommon/zfs_fletcher.c

```diff
@@ -397,6 +415,7 @@ fletcher_4_init(void)
 		ops->init(&zc);
 		do {
 			ops->compute(databuf, data_size, &zc);
+			ops->compute_byteswap(databuf, data_size, &zc);
```

Comment:
There's no real benefit to grouping these together. Changes we discussed for this method can be addressed in a separate PR.